app/initializers/fastboot.js

```diff
@@ -4,62 +4,18 @@ export default {
   name: "fast-boot",
 
   initialize: function(App) {
-    // Detect if we're running in Node. If not, there's nothing to do.
-    if (typeof document === 'undefined') {
-      var doc = new SimpleDOM.Document();
-      var domHelper = new Ember.HTMLBars.DOMHelper(doc);
-
-      domHelper.protocolForURL = function(url) {
+    // Detect if we're running in FastBoot mode. If not, there's nothing to do.
+    if (FastBoot) {
```

Comment:
should this be typeof check? Or will the ambient global always be present if this initializer runs?